docs/core-manage-versions.md

```diff
@@ -60,12 +60,15 @@ asdf latest <name> <version>
 
 ```shell
 asdf global <name> <version> [<version>...]
+asdf shell <name> <version> [<version>...]
 asdf local <name> <version> [<version>...]
 # asdf global elixir 1.2.4
 ```
 
 `global` writes the version to `$HOME/.tool-versions`.
 
+`shell` set the version to an environment variable `ASDF_$LANG_VERSION`, for the current shell session only.
```

Comment:
What about "set the version as an environment variable named `ASDF_${LANG}_VERSION`, for the current shell session only."?